libs/pika/execution/include/pika/execution/algorithms/ensure_started.hpp

```diff
@@ -115,8 +115,10 @@ namespace pika::execution::experimental {
             template <template <typename...> class Variant>
             using error_types =
                 pika::util::detail::unique_t<pika::util::detail::prepend_t<
-                    typename pika::execution::experimental::sender_traits<
-                        Sender>::template error_types<Variant>,
+                    pika::util::detail::transform_t<
```

Comment:
Could you update `value_types` as well (with a corresponding test)?

Reply:
As discussed, already tested here:
https://github.com/pika-org/pika/blob/5342bce01424dc1fcb78dec0578861b3a3fe7354/libs/pika/execution/tests/unit/algorithm_ensure_started.cpp#L104